pyttb/sptensor.py

```diff
@@ -671,12 +883,36 @@ def innerprod(
 
     def isequal(self, other: Union[sptensor, ttb.tensor]) -> bool:
         """
-        Exact equality for sptensors
+        Exact equality for sparse tensors, where all elements are exactly the
+        same in both tensors. Can be compared to another sparse tensor or
+        dense tensor.
 
         Parameters
         ----------
-        other:
-            Other tensor to compare against
+        Other tensor to compare against.
```

Comment:
The rendering in sphinx looks weird when just a description rather than the parameter name is provided. For consistency I think we want to keep the param name even if there is just one.
```suggestion
        other:
           Other tensor to compare against.
```